cmd/api/handlers/player/get_all.go

```diff
@@ -0,0 +1,21 @@
+package player
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
```

Comment:
Falta añadir los comentario para el swagger

Reply:
Gracias por las observaciones @DarcoProgramador,  tengo dudas de como están manejando los comentarios para swagger?, no veo ejemplos en el repo actual 

algo así seria?

```
  // GetAll retrieves all players from the repository.
  // @Summary Get all players
  // @Description Retrieves all players from the repository
  // @Tags players
  // @Produce json
  // @Success 200 {array} domain.Player
  // @Failure 500 {object} domain.AppError
  // @Router /players [get]
  // It returns an Array of domain.Player objects and an error.
  // If the player is not found in the repository, it returns a domain-specific not found error.
  // If there is a timeout error when accessing the repository, it returns a domain-specific timeout error.
  // For any other errors, it logs the error and returns a generic error.

```

Reply:
Fuentes: https://medium.com/julotech/implementing-swagger-in-go-projects-8579a5fb955

Reply:
resuelta